<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/auto.jpg</normaloff>image/auto.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;仿宋&quot;;
</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>500</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;04b_21&quot;;
font: 20pt &quot;04b_21&quot;;
color: rgb(0, 0, 0);
font: 20pt &quot;仿宋&quot;;</string>
        </property>
        <property name="text">
         <string>三维轨迹规划与运动控制系统</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_login">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>150</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;仿宋&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>开始登陆</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/password.png&quot;/&gt;密码：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QWidget" name="widget" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_register">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);font: 12pt &quot;仿宋&quot;;</string>
                </property>
                <property name="text">
                 <string>注册</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>image/rigester.png</normaloff>image/rigester.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_queren_login">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);font: 12pt &quot;仿宋&quot;;</string>
                </property>
                <property name="text">
                 <string>确认</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>image/login.png</normaloff>image/login.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);font: 12pt &quot;仿宋&quot;;</string>
                </property>
                <property name="text">
                 <string>退出</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>image/out.png</normaloff>image/out.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_2">
             <property name="placeholderText">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>76</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image: url(:/image/cug.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit">
             <property name="placeholderText">
              <string>Account</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/mana.png&quot;/&gt;用户名：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;04b_21&quot;;
</string>
          </property>
          <property name="text">
           <string>学号</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;04b_21&quot;;</string>
          </property>
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QLineEdit" name="lineEdit_8"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;04b_21&quot;;</string>
          </property>
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_4"/>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;04b_21&quot;;</string>
          </property>
          <property name="text">
           <string>性别</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" colspan="2">
         <widget class="QLineEdit" name="lineEdit_5"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;04b_21&quot;;</string>
          </property>
          <property name="text">
           <string>年龄</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;04b_21&quot;;</string>
          </property>
          <property name="text">
           <string>所在系</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLineEdit" name="lineEdit_7"/>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QPushButton" name="pushButton_queren">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 170, 255);
font: 12pt &quot;04b_21&quot;;</string>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
